Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -866,8 +876,12 @@ def enrich_offense_with_assets(client: Client, offense_ips: List[str]) -> List[D
 
     def get_assets_for_ips_batch(b: List):
         filter_query = ' or '.join([f'interfaces contains ip_addresses contains value="{ip}"' for ip in b])
-        return client.assets_list(filter_=filter_query)
+        try:
+            return client.assets_list(filter_=filter_query)
+        except Exception as e:
+            raise DemistoException(f'Error occurred during asset enrichment. Query: {filter_query}') from e
```

Comment:
How that impact the long-running? It would stop and when would be the next time it will run?

Reply:
It will not stop long running. It will log the error as the long-running execution loop is wrapped by
try:
 While True:
  - do long running
except Exception:
- log error
finally:
- sleep